src/components/Starred.jsx

```diff
@@ -0,0 +1,42 @@
+import { useSelector, useDispatch } from 'react-redux'
+import { Link } from 'react-router-dom'
+import starredSlice from '../data/starredSlice'
+import Movie from './Movie'
+import '../styles/starred.scss'
+
+const Starred = ({viewTrailer}) => {
+
+    const state = useSelector((state) => state)
+    const { starred } = state
+    const { clearAllStarred } = starredSlice.actions
+    const dispatch = useDispatch()
+
+  return (
+    <div className="starred" data-testid="starred">
+      {starred.starredMovies.length > 0 && (<div data-testid="starred-movies" className="starred-movies">
+        <h6 className="header">Starred movies</h6>
+        <div className="row">
+        {starred.starredMovies.map((movie) => (
+          <Movie 
+            movie={movie} 
+            key={movie.id}
+            viewTrailer={viewTrailer}
+          />
+        ))}
+        </div>
+
+        <footer className="text-center">
+          <button className="btn btn-primary" onClick={() => dispatch(clearAllStarred())}>Remove all starred</button>
+        </footer>
+      </div>)}
+
+      {starred.starredMovies.length === 0 && (<div className="text-center empty-cart">
```

Comment:
You can avoid this extra conditional if on line 16 you made something like this: `{starred.starredMovies.length > 0 ? <>RENDER_HTML_DATA</> : <>RENDER_NO_HMTL_FOR_EMPTY</>`